lib/game_team.rb

```diff
@@ -0,0 +1,51 @@
+require "csv"
+
+class GameTeam
+  attr_reader :game_id,
+              :team_id,
+              :hoa,
+              :result,
+              :settled_in,
+              :head_coach,
+              :goals,
+              :shots,
+              :tackles,
+              :pim,
+              :ppo,
+              :ppg,
+              :fowp,
+              :giveaways,
+              :takeaways
```

Comment:
These lines can be deleted since they relate to data we won't be using.

Reply:
Let's keep them for now. The spec harness requires a few of the columns that we won't be using for our data but are in the spec tests.

Reply:
Oh yeah, good call - didn't even think about it in terms of the spec harness.